apps/backend/src/main.ts

```diff
@@ -47,7 +48,12 @@ async function bootstrap() {
   app.use(
     session({
       secret: configService.get('JWT_SECRET') || generateDefault(),
-      saveUninitialized: false,
+      store: new (postgresSessionStore(session))({
+        conString: configService.getDbUrl(),
```

Comment:
The `postgresSessionStore` apparently takes a `conObject` option as well. Could you see if it is compatible with our `getDbConfig` object?